README.md

```diff
@@ -0,0 +1,165 @@
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 Awesome-books <a name="about-project"></a>
+
+
+
+**Awesome-books-project** is a project to showcase my html,CSS and javascript skills applied in bulding a project
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Add_header]**
+- **[Add_styling]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+
+- [Live Demo Link]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+web browser
+ 
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  cd my-folder
+  git clone https://github.com/Naledi-Dikgale/awesome-books
+```
+
+
+### Install
+
+Install this project with:
+
+```sh
+  cd awesome-books
+  npm install
+```
+
+### Usage
+
+To run the project, follow the instruction below:
+open index.html in your browser
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="Naledi Dikgale"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Naledi Dikgale**
+
+- GitHub: [@Naledi-Dikgale](https://github.com/Naledi-Dikgale)
+- LinkedIn: [naledi-dikgale-068423159](https://www.linkedin.com/in/naledi-dikgale-068423159/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Add_footer]**
+- [ ] **[Add_navbar]**
+- [ ] **[Add_images]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](project/issues).
+
+<p align="right">(<a href="#reahttps://github.com/Naledi-Dikgale">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project Please star it in my github page.
+
+<p align="right">(<a href="https://github.com/Naledi-Dikgale">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name=""></a>
+
+
+I would like to thank my teammates.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](/MIT.md) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```

Comment:
- [ ] it would be better to remove this it was only for you it shouldn't be in the readme file